omnipaxos/src/sequence_paxos/leader.rs

```diff
@@ -42,6 +42,7 @@ where
                 suffix: vec![],
                 stopsign: self.internal_storage.get_stopsign(),
             };
+            self.leader_state.reset_promises();
```

Comment:
This should not be needed since we create a new LeaderState before (on line 27).

Reply:
Can therefore also remove the reset_promises() function.